doc/man/8/rbd.rst

```diff
@@ -65,19 +65,21 @@ Parameters
      support for cloning and is more easily extensible to allow more
      features in the future.
 
-.. option:: --size size-in-M/G/T
+.. option:: -s size-in-M/G/T, --size size-in-M/G/T
```

Comment:
Options and Parameters sections use comma: -c / --conf, -p / --pool, etc.  The entire man page could use a face lift - I erred on the side of consistency.